SR2020/2020-05-22-virtual-competition-transition.md

```diff
@@ -0,0 +1,26 @@
+---
+to: sr2020-teams
+subject: Transitioning to a virtual competition
+---
+
+Hi all,
+
+In light of recent events, and the uncertainty it has brought, we're unfortunately cancelling the in-person competition for Student Robotics 2020.
+
+## Virtual Competition
+
+The virtual competition event will take place over a few weekends in July, with a few weekends of league matches, and a knockout final.
+
+As we said in the previous email, we aim to keep the virtual competition rules as similar to Two Colours as possible so teams can use the code they have been working on since Kickstart.
```

Comment:
This is still talking about re-using code, yet with the mode to Python 3 and the other (subtle but still present) changes to the API it feels slightly misleading to say that.

Is there something we can say which is a bit more general? Perhaps just say something like "easily build on the work they have done since Kickstart"?